events/wc_order_events.txt

```diff
@@ -906,31 +906,130 @@ narrative_event = {
 		name = EVTOPTB_WCORD_406
 	}
 }
+# The Creation of Blood Knights Order
+namespace = WCBKO
+narrative_event = {
+	id = WCBKO.01
+	title = EVTTITLE_WCBKO_1
+	desc = EVTDESC_WCBKO_1
+	picture = GFX_evt_sunwell
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+	
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+	show_root = yes
+	
+	immediate = {
+		create_character = {
+			random_traits = yes
+			dynasty = none
+			female = no
+			religion = cult_of_sunwell
+			culture = high_elf
```

Comment:
I'm Sin'dorei, the order leader is Quel'dorei.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81822316-fdb3fc00-9543-11ea-8761-7eb5b788533d.png)
![image](https://user-images.githubusercontent.com/46384992/81822411-1de3bb00-9544-11ea-92bb-5da14c73ecc5.png)

</details>

```suggestion
			culture = ROOT
```